spec/index.md

```diff
@@ -780,17 +780,31 @@ Given a query (the `content` map from the request body) `Q`, a response `R`, and
               status: "replied",
               reply: R.reply,
               timestamp: T,
-              request_id: hash_of_map(Q)}))
+              request_id: hash_of_map({
```

Comment:
We already have a section of request id in the [specification](https://internetcomputer.org/docs/current/references/ic-interface-spec/#request-id).

> When signing requests or querying the status of a request (see [Request status](https://internetcomputer.org/docs/current/references/ic-interface-spec/#state-tree-request-status)) in the state tree, the user identifies the request using a request id, which is the [representation-independent hash](https://internetcomputer.org/docs/current/references/ic-interface-spec/#hash-of-map) of the content map of the original request. A request id must have length of 32 bytes.

When the user sign the query request,  it still use the original request id(with ingress expiry). Shall we distinguish these two things in the specification to avoid confusion? For example, give this field a different name in the query response? What about `query_id`?

Reply:
Sorry that I just find I commented on an outdated line.

It is clear that ingress_expiry is explicitly ignored. Now I am fine with keeping request_id or changing to another name.

Reply:
Edit: Resolve it as the comment is outdated.